docs/formation/governance/working-groups/duplication-reduction/duplication-reduction-charter.md

```diff
@@ -0,0 +1,67 @@
+# Duplication Reduction Working Group
+
+This [WG] facilitates the research of existing frameworks for the Common Cloud Controls project community.
+
+## Mission
+
+This working group will evaluate the tools, assets, and partners that exist in the cloud controls space that can help reduce the effort required to meet the overall CCC goals for the CCC contributors and consumers. 
+
+Our goal is to ensure the CCC efforts do not duplicate pre-existing standards, and instead are net new contributions to the broader ecosystem aligned to the specific needs of the financial services industry.
+
+## Approach
+
+This work will span 2 main outputs:
+
+### 1 — Investigate existing frameworks
+
+The first step is to identify a set of existing frameworks and capabilities that we should investigate (eg. NIST, ISO, OSCAL, MITRE, etc) against the CCC goals. 
+
```

Comment:
Just had a quick look and couldn't find anything that documents the goals of CCC. The steering group charter has mission/objectives (https://github.com/finos/common-cloud-controls/blob/main/docs/governance/steering/charter.md) are the project goals documented anywhere?

Reply:
Perhaps `CCC [WG] Goals` would suit better?